en/docs/deploy/deployment-guide.md

```diff
@@ -450,46 +440,38 @@ Identity Server endpoints. The `hostName` should be resolved to the Load Balance
 
 ## Enable artifact synchronization
 
-To enable synchronization for runtime artifacts, you must have a shared file system. You can use one of the following
- depending on your environment.
- 
-   -   **Network File System (NFS)**: This is one of the most commonly known shared file systems and can be used 
-   in a Linux environment.
-   -   **Server Message Block (SMB) file system**: This can be used in a Windows environment.
-   -   **Amazon EFS**: This can be used in an AWS environment.
+You must have a shared file system to enable synchronization for runtime artifacts. You can use one of the following depending on your environment.
+
+   - **Network File System (NFS)**: This is one of the most commonly known shared file systems and can be used in a Linux environment.
+   - **Server Message Block (SMB) file system**: This can be used in a Windows environment.
+   - **Amazon EFS**: This can be used in an AWS environment.
 
-Once you have chosen a file system, 
+Once you have chosen a file system:
 
 1. Mount it in the nodes that are participating in the cluster.
 2. Create two directories called `Deployment` and `Tenants` in the shared file system.
-3. Create a symlink from the `<IS_HOME>/repository/deployment` path to the `Deployment` directory of the shared 
-file system that you created in the previous step.
-4. Create a symlink from the `<IS_HOME>/repository/tenants` path to the `Tenants` directory of the shared file 
-system that you created.
+3. Create a symlink from the `<IS_HOME>/repository/deployment` path to the `Deployment` directory of the shared file system you created in the previous step.
```

Comment:
Isn't a symlink mainly a Linux-specific thing? What's our story with other operating systems?

Reply:
We will look into these updates throuhh [1].

[1] https://github.com/wso2/product-is/issues/15117